apps/web/package.json

```diff
@@ -69,6 +69,7 @@
     "async": "^3.2.4",
     "bcryptjs": "^2.4.3",
     "classnames": "^2.3.1",
+    "discourse-sso": "^1.0.5",
```

Comment:
Using this dependency to do the heavy lifting behind the logic to verify a login request from Discourse.